config/deploy/qa.rb

```diff
@@ -2,7 +2,7 @@
 
 set :rails_env, :production
 set :bundle_without, %w[development test].join(' ')
-set :branch, 'qa'
+set :branch, 'createvariables'
```

Comment:
You don't want to push this to qa.

Reply:
It is incorrect to change the branch name in qa.  QA's branch name should remain "qa".  When you change it to do test deploys, you only change the branch name LOCALLY, and do not commit that change.

Reply:
going to get rid of this branch